cluster-scope/base/core/namespaces/aap/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    openshift.io/cluster-monitoring: "true"
+  name: aap
```

Comment:
I know that I am going to both misread and mistype this as `app` all the time, but I see that this is the canonical namespace name for this operator so I guess we keep it :smile: .